sandbox/src/index.ts

```diff
@@ -9,6 +9,26 @@ import { runJava } from './run/java.js';
 
 export const timeoutSeconds = 30;
 
+const RunResultKind = z.enum(["CompileFailed", "TimeLimitExceeded", "Completed", "SandboxError"]);
```

Comment:
I know there's some duplicate declaration of these types across projects, but didn't want to try to make a spot for shared types as part of this change.